### PR TITLE
VZ-10270: add the verrazzano namespace label to the verrazzano-capi namespace

### PR DESCRIPTION
**Pull request by mbenmclean**
Add the verrazzano.io/namespace label to the verrazzano-capi namespace

### CODE REVIEW

**Comment by kminder** [comment]
Test?

**Comment by mbenmclean** [comment]
> Test?

added a test

**Review by adiforluls — DISMISSED** [review]

File: tests/e2e/pkg/common.go [hunk 1]
```diff
@@ -860,3 +860,17 @@ func GetNodeCountInCluster(kubeconfigPath string) (int, error) {
 	}
 	return len(nodes.Items), nil
 }
+
+// DoesNamespaceHasVerrazzanoLabel checks whether the namespace has the verrazzano.io/namespace label
+func DoesNamespaceHasVerrazzanoLabel(ns string) bool {
```
Comment (kminder) [inline tests/e2e/pkg/common.go:865]: This type of test function should almost always return an error also.
This helps make the test report easier to read.

**Review by kminder — APPROVED** [review]
Approved.

**Pull request merged**